applications/vanilla/models/class.discussionmodel.php

```diff
@@ -289,10 +292,9 @@ public function get($Offset = '0', $Limit = '', $Wheres = '', $AdditionalFields
         }
 
         // Get preferred sort order
-        $SortField = self::GetSortField();
+        $orderFields = $this->getOrderFields();
 
-        $this->EventArguments['SortField'] = &$SortField;
-        $this->EventArguments['SortDirection'] = c('Vanilla.Discussions.SortDirection', 'desc');
+        $this->EventArguments['OrderFields'] = &$orderFields;
```

Comment:
This breaks backwards compatibility for any plugin manipulating this, and I don't see any good reason for that.